pages/developers/license.md

```diff
@@ -1,801 +0,0 @@
-## License
-Copyright (C) 2023 dYdX Trading Inc.
```

Comment:
to confirm, are we deleting this because we are adding this text to the open_source_repo... below?

Reply:
Yes!